mvn-defaults/pom.xml

```diff
@@ -14,7 +14,8 @@
   ~ limitations under the License.
   -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
It may be possible to include default executions here, as all modules in the project inherit from this one.

Reply:
Yes, I was thinking the same until I got to thinking about the configuration portion of modules where we may want to have a different code coverage min and max settings.  For example the following is in place at ea. module level, but the overall reporting gets pulled from the maven-defaults for an aggregate view.  I'm open to configuring for all in one place if we think that makes more sense for all the modules however.

```xml
<execution>
  <id>jacoco-check</id>
  <phase>verify</phase>
  <goals>
      <goal>check</goal>
  </goals>
  <configuration>
      <rules>
          <rule>
              <element>BUNDLE</element>
              <limits>
                  <limit>
                      <counter>INSTRUCTION</counter>
                      <value>COVEREDRATIO</value>
                      <minimum>0.00</minimum>
                      <maximum>1.00</maximum>
                  </limit>
              </limits>
          </rule>
      </rules>
  </configuration>
</execution>
```